src/metrics.rs

```diff
@@ -2159,6 +2235,23 @@ impl GooseMetrics {
 
         Ok(())
     }
+
+    /// Records number of users for a current second.
+    ///
+    /// This is called from [`GooseAttack::sync_metrics()`] and the data
+    /// collected is used to display users graph on the HTML report.
+    pub(crate) fn record_users_per_second(&mut self) {
+        if let Some(starting) = self.starting {
```

Comment:
Just FYI: there are two upcoming changes that will impact this: 1) it will be possible to adjust the # of users of a running load test through the controller, so this value will be able to change outside of starting, and 2) in #365 it will become possible to shape the traffic to increase and decrease so again the value will be able to change outside of starting

Reply:
I *think* that this should still work, but we will test it.